variables.tf

```
@@ -184,6 +184,12 @@ variable "terraform_organization" {
   description = "The Terraform Enterprise organization to create the workspace in"
 }
 
+variable "terraform_ssh_key_id " {
+  type        = string
+  default     = null
+  description = "The ID of an SSH key to assign to the workspace"
+}
```

```suggestion
variable "ssh_key_id " {
  type        = string
  default     = null
  description = "The SSH key ID to assign to the workspace"
}
```

Hey @marwinbaumannsbp, you can shorten the variable name as it's implied it's for the workspace because it's in the workspace module. It would need a prefix if it was for something other than the workspace IMO.